PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java

```diff
@@ -251,8 +250,11 @@ public void finished(boolean aborted, int errors, int warnings, CompileContext c
                                         null,
                                         postEventProperty);
 
-                                //FIXME!!! Sometimes, the submission message window not ready
-                                //   HDInsightUtil.showInfoOnSubmissionMessageWindow(project, String.format("Info : Build %s successfully.", artifact.getOutputFile()));
+                                DefaultLoader.getIdeHelper().executeOnPooledThread(() -> {
+                                    HDInsightUtil.showInfoOnSubmissionMessageWindow(
+                                            project,
+                                            String.format("Info : Build %s successfully.", artifact.getOutputFile()));
```

Comment:
artifact or its output file could be null if we have a incorrect compile result.